dynamic-layers/openembedded-layer/recipes-support/landing-zone/landing-zone_0.3.0.bb

```diff
@@ -17,6 +17,11 @@ S = "${WORKDIR}/git"
 EXTRA_OEMAKE += "DEBUG=y"
 SECURITY_STACK_PROTECTOR = ""
 
+do_configure_prepend() {
```

Comment:
@Asiderr use gitsm instead of git in SRC_URI to fetch modules

Reply:
Fixed